src/components/RouterLink.tsx

```diff
@@ -1,11 +1,14 @@
 import { Link } from 'react-router-dom';
 import styled, { css } from 'styled-components';
 
+import { COLORS } from '@theme';
+
 interface RouterLinkProps {
   fullwidth?: boolean;
 }
 
 const StyledRouterLink = styled(Link)<RouterLinkProps>`
+  color: ${COLORS.BLUE_SKY} !important;
```

Comment:
use ampersands (`&&&`) instead of `!important`

https://styled-components.com/docs/faqs#how-can-i-override-styles-with-higher-specificity